src/settings-ui/Microsoft.PowerToys.Settings.UI/Views/GeneralPage.xaml

```diff
@@ -149,6 +149,12 @@
                     IsEnabled="{Binding AutoUpdatesEnabled}"
                     />
 
+            <Button x:Uid="GeneralPage_UpdateNow"
+                    Style="{StaticResource AccentButtonStyle}"
+                    Foreground="White"
```

Comment:
You should reference a brush here. I think otherwise it can be problematic on theme change.